ksql-functional-tests/src/main/java/io/confluent/ksql/test/tools/FakeKafkaService.java

```diff
@@ -59,16 +59,28 @@ void writeRecord(final String topicName, final FakeKafkaRecord record) {
     return ImmutableList.copyOf(topicData.get(topicName));
   }
 
-  private void ensureTopicExists(final String topicName) {
+  public void ensureTopicExists(final String topicName) {
```

Comment:
nit: I think the convention is `ensure` will create it if it doesn't exist. maybe instead name it `requireTopicExists` to be inline with `requireNonNull`